src/arch/x64/devices/mod.rs

```diff
@@ -2,6 +2,7 @@ pub mod serial;
 pub mod pic;
 pub mod rtc;
 pub mod pit;
+pub mod high_precision_timer;

```

Comment:
To keep the theme of three letter device names, do you think renaming the module to `hpt` would work?

Reply:
It would if you really want to keep that theme.

I'm generally of the opinion that acronyms like that just make the code harder to read and aren't worth it.

Reply:
That's fair. Let's keep it full length then.